include/openssl/crypto.h.in

```diff
@@ -86,6 +86,9 @@ int CRYPTO_THREAD_unlock(CRYPTO_RWLOCK *lock);
 void CRYPTO_THREAD_lock_free(CRYPTO_RWLOCK *lock);
 
 int CRYPTO_atomic_add(int *val, int amount, int *ret, CRYPTO_RWLOCK *lock);
+int CRYPTO_atomic_or(uint64_t *val, uint64_t op, uint64_t *ret,
+                     CRYPTO_RWLOCK *lock);
+int CRYPTO_atomic_load(uint64_t *val, uint64_t *ret, CRYPTO_RWLOCK *lock);
```

Comment:
Is there a reason to make those functions public?

Reply:
I pondered this myself. We made the other "atomic" function public, and other similar threading type functions, so I decided that if we thought those ones were generally useful, then this one is too. OTOH, I don't have a strong preference if you'd rather them not be.